### ISSUE TITLE
AzDO Issue #3481

### ISSUE THREAD

**Issue by net-helix[bot]** [dotnet/dnceng#1180]
[Azure Boards WorkItem](https://dev.azure.com/dnceng/7ea9116e-9fac-403d-b258-b31fcf1bb293/_workitems/edit/3481)

**Comment by missymessa** [comment]
@garath or @riarenas do you recall if this work was done?

**Comment by garath** [comment]
I do not believe it was done, but I do not recall. The history here indicates it was not done.

**Comment by missymessa** [comment]
Linked issue closed. Closing.
